.storybook/test-data.js

```diff
@@ -467,8 +467,28 @@ const state = {
         isEns: true,
       },
     ],
-    contractExchangeRates: {
-      '0xaD6D458402F60fD3Bd25163575031ACDce07538D': 0,
+    marketData: {
```

Comment:
Given how much of this PR relates to the breaking changes in token rate controller state (now published as assets-controller version 30), it might be cleaner to do a dedicated PR for that.